### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Ivan-Orlov** [avocode/avocode#2205]
---
* **Version**: 3.5.6
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 199840
---
```
TypeError: Failed to execute 'shaderSource' on 'WebGLRenderingContext': parameter 1 is not of type 'WebGLShader'.
    at _glShaderSource (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:8:652607)
    at TP (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:12:192543)
    at E6 (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:13:195479)
    at TK (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:14:591113)
    at sg (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:20:64884)
    at xj (C:\Users\User\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\@avocode\rendering-emscripten\rendering-asm.js:17:5
```

**Comment by crashmajstr** [comment]
Hello @Ivan-Orlov  ,
would you be able to describe when exactly did this error happen? Do you remember what were you doing when the error popped out?

Were you trying to open a specific design? If so, does the error pop up everytime you try to open the same design?

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 which could fix this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Let me know if the problem still persists or not. Thank you
